essentials.yml

```diff
@@ -1,7 +1,30 @@
 ---
+packaging:
+  config:
+    bundle:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      vendor: "Jenkins project"
+      title: "Jenkins Integration Tests WAR"
+      description: "Jenkins WAR for integration tests"
+    war:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      source:
+        dir: "."
```

Comment:
Should use the incrementals version instead IMHO.
CC @jglick is there an easy way to retrieve it from `maybePublishIncrementals()` inside the Pipeline?

Reply:
That would simply be the version of the module as reported by Maven, which `Jenkinsfile` does also retrieve. But anyway I do not think that is what you want to do. Would suffice to just point to a filepath (glob) of the WAR file that was just built, right?

Reply:
> Would suffice to just point to a filepath (glob) of the WAR file that was just built, right?

Yes if it is the incrementals snapshot with uploaded dependencies.
Unlikely if it is a build with non-deployed SNAPSHOT dependencies, because PCT magic for that is not very stable (my work to stabilize it was cancelled).

Reply:
> a build with non-deployed SNAPSHOT dependencies

Do I understand correctly that you are referring to a local development scenario? But then there is no incremental version anyway—the WAR will be a snapshot.